openshift_resources/oadp/data_protection_application.py

```diff
@@ -1,17 +1,516 @@
-import logging
-import wrapper_constants.resources as r
+# generated by datamodel-codegen:
+#   filename:  k8s_swagger.json
+#   timestamp: 2022-10-29T09:10:36+00:00
 
-from ocp_resources.resource import NamespacedResource
+from __future__ import annotations
+from datetime import datetime
+from enum import Enum
+from typing import Any, Dict, List, Optional
 
-logger = logging.getLogger(__name__)
+from pydantic import BaseModel, Field, conint, constr
+from kubernetes import config
+from openshift.dynamic import DynamicClient
+import resources.io.k8s.apimachinery.pkg.apis.meta.v1 as v1
 
+from openshift_resources.resource import BaseResource
 
-class DataProtectionApplication(NamespacedResource):
-    api_group = r.ApiGroups.OADP_API_GROUP.value
 
-    def wait_for_successful_reconciliation(self):
-        self.wait_for_condition(condition=r.Condition.CONDITION_RECONCILED.value, status=r.Status.SUC_RECONCILED.value)
+class CreationSecret(BaseModel):
```

Comment:
redundant, defined here already:
https://github.com/oadp-qe/openshift-resources/blob/main/src/resources/io/k8s/api/core/v1.py#L1342

Reply:
Cannot find it 
https://pkg.go.dev/search?q=CreationSecret&m=symbol
maybe you mean another class?
@mperetzred